app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -322,6 +322,23 @@ public void setBottomCardVisibility(@UploadPage int page) {
         }
     }
 
+    /**
+     * Only show the subtitle ("For all images in set") if multiple images being uploaded
+     * @param imageCount Number of images being uploaded
+     */
+    @Override
+    public void updateSubtitleVisibility(int imageCount) {
+        TextView categoriesSubtitle = viewFlipper.findViewById(R.id.categories_subtitle);
+        if (categoriesSubtitle != null) {
+            categoriesSubtitle.setVisibility(imageCount > 1 ? View.VISIBLE : View.GONE);
+        }
+
+        TextView licenseSubtitle = viewFlipper.findViewById(R.id.license_subtitle);
+        if (licenseSubtitle != null) {
+            licenseSubtitle.setVisibility(imageCount > 1 ? View.VISIBLE : View.GONE);
+        }
```

Comment:
You will see a series of @BindView on top of this file. There http://jakewharton.github.io/butterknife/ library helps us to simplify our view binding. You can add these lines to the end of bindings, comewhere like line 120

    // Subtitle
    @BindView(R.id.categories_subtitle) TextView categoriesSubtitle;
    @BindView(R.id.license_subtitle) TextView licenseSubtitle;

And remove these lines: 
TextView licenseSubtitle = viewFlipper.findViewById(R.id.license_subtitle);
TextView categoriesSubtitle = viewFlipper.findViewById(R.id.categories_subtitle);

By this way we do not create these variables each time this method is called, and also our code will be consistent in terms of style.

Reply:
I have made the requested change.